docs/quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quickstart
+
+## Run bblfsh and gitbase Dependencies
+
+It is recommended to read about `bblfsh` and `gitbase` from its own documentation, but here is a small guide about how to run both easily:
+
+Launch [bblfshd](https://github.com/bblfsh/bblfshd) and install the drivers. More info in the [bblfshd documentation](https://doc.bblf.sh/user/getting-started.html):
+
+```bash
+$ BBLFSHD_CONTAINER=bblfsh
+$ BBLFSHD_IMAGE=bblfsh/bblfshd
+$ docker run --privileged
+    --publish 9432:9432
+    --volume /var/lib/bblfshd:/var/lib/bblfshd
+    --name ${BBLFSHD_CONTAINER}
+    ${BBLFSHD_IMAGE}
+$ docker exec -it ${BBLFSHD_CONTAINER}
+    bblfshctl driver install --recommended
+```
+
+[gitbase](https://github.com/src-d/gitbase) will serve git repositories, so it is needed to populate a directory with them:
+
+```bash
+$ REPOS_PATH=~/gitbase/repos
+$ mkdir -p ${REPOS_PATH}
+$ git clone git@github.com:src-d/go-git-fixtures.git ${REPOS_PATH}/go-git-fixtures
+```
+
+Install and run [gitbase](https://github.com/src-d/gitbase):
+
+```bash
+$ REPOS_PATH=~/gitbase/repos
+$ GITBASE_CONTAINER=gitbase
+$ BBLFSHD_CONTAINER=bblfsh
+$ GITBASE_IMAGE=dpordomingo/gitbase:latest
```

Comment:
This is needed untill https://github.com/src-d/gitbase/issues/262 resolved

I would put a comment right in the doc, if upstream issue does not get resolved quickly.

Reply:
I'll do so